admin-dev/themes/new-theme/scss/pages/product/product_page.scss

```diff
@@ -109,6 +109,105 @@ $product-page-padding-bottom: 80px !default;
   }
 
   // Components
+  .product-header-v2 {
+    display: flex;
+    flex-direction: row;
+    gap: 25px;
+    padding: 0.625rem;
+    margin-bottom: 1rem;
+    background: $white;
+    @include border-radius(0.375rem);
+    box-shadow: 0 0 12px rgba(0, 0, 0, 0.15);
+
+    > * {
+      flex-grow: 1;
+    }
+
+    $product-cover-size: 110px;
+
+    .product-header-cover {
+      width: $product-cover-size;
+      max-width: $product-cover-size;
+      height: $product-cover-size;
+      max-height: $product-cover-size;
```

Comment:
Max width and max-height are useless, the width and height can't change if they are fixed

Why no just use img-fluid? The image should adapt depending on his width but maybe there are some PHP stuff that force the size anyway

Reply:
Actually, since I'm in a flex container the max-width and height are the most important Without it the div was too large despite the presence of width/height

Reply:
As I already explained to @Hlavtox the image is already in `img-fluid` This class doesn't target the image it targets its div container which must be restricted and keep a fixed size (the flex container tries to make it larger if I don't restrict it)

Reply:
`flex-grow:0;`

Reply:
Thanks @Hlavtox `flex-grow: 0;` works nicely indeed!

I just left a max-width though because the image I use is still wider than the expected cover. We don't have an image type with 110px size and I don't think it's worth creating a new image type for this.